iree/test/e2e/models/bert_encoder_unrolled_fake_weights.mlir

```diff
@@ -1,1120 +1,1120 @@
 // MobileBert encoder model with placeholder weights, for testing.
 
 module {
-  flow.variable @"__iree_flow_bert/embeddings/FakeLayerNorm/beta" dense<1.0> : tensor<512xf32> attributes {noinline, sym_visibility = "private"}
```

Comment:
Is this a permanent change or is it just to temporary unblock the test?

Reply:
Could be permanent. This should not be used to get perf numbers though. I think the tensorflow integration tests is what is used to get performance numbers right?

Reply:
the case used for perf AFAIK doesnt have all constants being splat constants. I think that was explicitly done to make the e2e/models/ more readable. This is functionally close, but point taken. I will revert the change to the test once I fix that issue.

Reply:
got it, thanks for explaining

Reply:
Yes, they are splat constants in e2e/models, so it's called _faked_weight_. Otherwise, the file size will be very large.